notify_send_tests.py

```diff
@@ -62,8 +62,9 @@
 
 
 def new_notification(source='source', message='message', icon='icon.png',
-                     timeout=5000, transient=True, urgency='normal'):
-    return Notification(source, message, icon, timeout, transient, urgency)
+                     desktop_entry="desktop", timeout=5000, transient=True,
```

Comment:
For consistency, I suggest replacing `"` with `'`, i.e. please change `"desktop"` to `'desktop'`.